pkg/block/gs/adapter.go

```diff
@@ -26,10 +25,7 @@ const (
 )
 
 var (
-	ErrMismatchPartETag    = errors.New("mismatch part ETag")
```

Comment:
Did we lose our verification that content matches original?

Reply:
OK yes S3 sends etags for the parts with the complete request with the expectation that we will through an error in the case of a mismatch

Reply:
Yeah, both validations are critical to the MPU functionality. Reliability is the top concern.

Reply:
OK so I had to return the list to do this- I did add a filter so that previous calls that time out will not effect the next call